Library/Homebrew/test/dev-cmd/bottle_spec.rb

```diff
@@ -194,3 +194,104 @@ def stub_hash(parameters)
     )
   end
 end
+
+describe "brew bottle --merge", :integration_test do
+  it "adds the bottle block to a fomrula that has none" do
```

Comment:
```suggestion
  it "adds the bottle block to a formula that has none" do
```

Reply:
`RSpec/ExampleLength: Example has too many lines [83/75].` 

Can add to `Library/Homebrew/test/.rubocop_todo.yml`